doc/source/glossary.rst

```diff
@@ -54,6 +145,84 @@ Glossary
        Any sequence that can be interpreted as an ndarray.  This includes
        nested lists, tuples, scalars and existing arrays.
 
+   array scalar
+      For uniformity in handling operands, NumPy treats
+      a :doc:`scalar <reference/arrays.scalars>` as an array of zero
+      dimension.
+
+
+   attribute
+       A property of an object that can be accessed using ``obj.attribute``,
+       e.g., ``shape`` is an attribute of an array::
```

Comment:
As far as I could tell there was one entry with 4 spaces, all the rest were 3. I would prefer to use 3 since then the content is more clearly delineated. Making the document 4 for the label and 8 for the content would require a reformat to keep line length down, and could be done in a future PR if desired.

Reply:
or were you refering to the content indenting 6 and 7, like this example?

Reply:
reformatted content, including docstrings. At some point we should push the whole document in 1 space so that it has 4 spaces for the labels and 8 for the content, not 3 and 7.

Reply:
I'm talking about how "A property" (155)  and "For uniformity" (149) have different indent levels.

I'd prefer to use four spaces everywhere, but that's fine for a follow-up PR.